app/uk/gov/hmrc/individualsbenefitsandcreditsapi/controllers/CommonController.scala

```diff
@@ -63,14 +66,17 @@ abstract class CommonController @Inject()(
       ErrorInvalidRequest(br.getMessage).toHttpResponse
     }
     case e: IllegalArgumentException => {
+      Logger.warn("Controllers IllegalArgumentException encountered")
       auditHelper.auditApiFailure(correlationId, matchId, request, url, e.getMessage)
       ErrorInvalidRequest(e.getMessage).toHttpResponse
     }
     case e: InternalServerException => {
+      Logger.warn("Controllers InternalServerException encountered")
       auditHelper.auditApiFailure(correlationId, matchId, request, url, e.getMessage)
       ErrorInternalServer("Something went wrong.").toHttpResponse
     }
-    case e => {
+    case e: Exception => {
+      Logger.warn("Controllers InternalServerException encountered")
```

Comment:
General exception, not InternalServerException

Reply:
OOPs